AlphaWallet/Core/Types/RealmConfiguration.swift

```diff
@@ -59,3 +48,25 @@ struct RealmConfiguration {
     }
 
 }
+
+extension FileManager {
+    
+    func removeAllItems(directory: URL) {
+        do {
+            let urls = try contentsOfDirectory(at: directory, includingPropertiesForKeys: nil, options: .skipsHiddenFiles)
```

Comment:
Verified this?

Reply:
Oh, actually `removeItem(at:)` removes recursively right? Then why not just `removeItem(at: directory)`?

Reply:
`removeItem(at:)` with `directory` param  will remove directory `testSuiteWalletsForWalletAddresses`, it may cause error, as it creates only once at apps launch, and execution can enter into function `func migratedWalletAddressesStore(userDefaults: UserDefaults) -> WalletAddressesStoreType` several times during testing. It may cause error.